### PR TITLE
Added v5 and apiCall to "options"

### PR DESCRIPTION
**Pull request by danidomen**

### CODE REVIEW

File: lib/doofinder_api.php [hunk 1]
```diff
@@ -168,11 +173,19 @@ private function apiCall($params){
         $httpCode = curl_getinfo($session, CURLINFO_HTTP_CODE);
         curl_close($session);
 
+        var_dump($response);
```
Comment (ecoslado) [inline lib/doofinder_api.php]: Maybe you missed this.

**Comment by danidomen** [comment]
Ohhh, so sorry Kike! I was missed the line. Now is fixed :)

**Comment by ecoslado** [comment]
@danidomen works in Prestashop Module and has implemented a wrapper for options call. @JoeZ99, could you  have a look and merge?

**Comment by JoeZ99** [comment]
@danidomen, txs for the PR.

I've moved a few things around just for cosmetic.

**Pull request merged**